src/bridging/BridgingTypes.ts

```diff
@@ -5280,6 +5282,7 @@ const typeMap: any = {
     ], false),
     "FindIntentsByContextAgentRequestPayload": o([
         { json: "context", js: "context", typ: r("ContextElement") },
+        { json: "resultType", js: "resultType", typ: u(undefined, "") },
```

Comment:
Just for my own understanding, I was wondering if `typ: u(undefined, "")` violates the 'no unions of primitive types' guidelines from this PR https://github.com/finos/FDC3/pull/1200 , but I assume this is how optional parameters are expressed here, and this is fine because it's really just an optional string.

Reply:
I know its fine in this case as we already have optional parameters elsewhere and those can generally be implemented via alternative function signatures in other languages. I'd probably also exclude null from the set of primitive types when considering this as most languages have someway of supporting that (e.g. `Nullable<int> i = null;` in C#, or using the `Integer` class in Java instead of int.